VideoLocker/src/main/java/org/edx/mobile/view/adapters/CourseDiscussionResponsesAdapter.java

```diff
@@ -225,8 +226,15 @@ public void onException(Exception ex) {
     }
 
     private void bindNumberResponsesView(NumberResponsesViewHolder holder) {
-        holder.numberResponsesOrCommentsLabel.setText(holder.numberResponsesOrCommentsLabel.getResources().getQuantityString(
-                R.plurals.number_responses_or_comments_responses_label, discussionThread.getCommentCount(), discussionThread.getCommentCount()));
+        int responsesCount = discussionThread.getResponseCount();
+        if (responsesCount < 0) {
+            // The responses count is not available yet, so hide the view.
+            holder.numberResponsesOrCommentsLabel.setVisibility(View.GONE);
```

Comment:
Yes, the response count is not available in the thread list query. I calculate it from the count fields that are provided when querying the endorsed and non-endorsed responses. See [MA-1212][].

[MA-1212]: https://openedx.atlassian.net/browse/MA-1212